tests/integration/test_error_injection.py

```diff
@@ -99,6 +99,13 @@ def test_dynamodb_write_error_injection():
     config.DYNAMODB_WRITE_ERROR_PROBABILITY = 1.0
     assert_raises(ClientError, table.put_item, Item={PARTITION_KEY: short_uid(), 'data': 'foobar123'})
 
+    # with a probability of 1, BatchWriteItem does not throw errors but logs UnprocessedItems
```

Comment:
`BatchWriteItem` throws `ProvisionedThroughputExceededException` if ALL items in Batch are Throttled